core/src/main/java/com/scalar/db/storage/jdbc/query/SimpleSelectQuery.java

```diff
@@ -233,10 +248,13 @@ public void bind(PreparedStatement preparedStatement) throws SQLException {
 
     for (Conjunction conjunction : conjunctions) {
       for (ConditionalExpression condition : conjunction.getConditions()) {
-        if (!condition.getColumn().hasNullValue()) {
+        if ((condition.getOperator().equals(Operator.LIKE)
+            || condition.getOperator().equals(Operator.NOT_LIKE))) {
+          binder.bindLikeClause((LikeExpression) condition);
+        } else if (!condition.getColumn().hasNullValue()) {
```

Comment:
This isn't directly related to this PR, but do we need to bind a NULL value when `condition.getColumn().hasNullValue()` returns true, similar to what `PreparedStatementBinder` does?

Reply:
We don't need bindings for the columns with null values in conjunctions because the string `IS (NOT) NULL` is used for `Operator.IS(_NOT)_NULL`. But I'm not sure we should also use bind markers for them. Do you have any idea about that?

Reply:
Yeah, that makes sense. We don't need to bind null values in conjunctions. Thanks.